### PR TITLE
guide update: phantomjs is a prereq for testing

### PR DESCRIPTION
**Pull request by stefanspicer**

### CODE REVIEW

File: doc/guides/4 - Refinery Extensions/3 - Testing.textile [hunk 1]
```diff
@@ -26,6 +26,12 @@ And run
 $ bundle install
 </ruby>
 
+You will need to install phantomjs from http://phantomjs.org/ or through your OS package manager.
+
+<shell>
+$ sudo apt-get install phantomjs
+</shell>
+
```
Comment (parndt) [inline doc/guides/4 - Refinery Extensions/3 - Testing.textile]: Nice.. could you document that apt-get is for Linux and add these other examples please:

```
OS X with homebrew:
<shell>
$ brew install phantomjs
</shell>

With npm:
<shell>
$ npm install -g phantomjs
</shell>
```

Comment (stefanspicer) [inline doc/guides/4 - Refinery Extensions/3 - Testing.textile]: I can do that for sure.  Is there a good way to do it without taking up 3/4 of the screen for a simple little dependency install?

screen shot 2016-05-10 at 9 18 20 pm

Comment (parndt) [inline doc/guides/4 - Refinery Extensions/3 - Testing.textile]: Maybe, but this is potentially the most useful way?  If you can think of a more succinct form I'm keen.

**Comment by stefanspicer** [comment]
That's a little more compact.

**Comment by parndt** [comment]
Thanks!

**Comment by simi** [comment]
:tada:

**Pull request merged**